chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+# 31 Usage Profile section
+
+## 31.1 Usage Scope field <a name="31.1"></a>
+
+### 31.1.1 Description
+
+Software packages are sometimes generated with specific Build conditions based on package supplier's license conditions and intended usage in addition to original software source code. In those cases, it is preferred for the package supplier to clearly state in the SPDX document the intended usage of the package as envisioned when setting the Build conditions, and/or the inspection conditions for packages may also have to be clearly stated for each package.
+
+With this "Usage Scope" field, the intended usage assumed by the package supplier is appended to the generated package in a manner that supplements those conditions contained in the source code.
+
+
+Table 31.1 — Metadata for the usage scope field
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..* |
+| Format | ["DocumentRef-"[idstring]":"]SPDXID \<single line of text\> \| `NONE` \| `NOASSERTION`<br>where "DocumentRef-"`[idstring]`":" is an optional reference to an external SPDX document as described in [6.6](document-creation-information.md#6.6)<br>where `SPDXID` is a string containing letters, numbers, `.` and/or `-`. to designate SPDX document/package/file as described in [6.3](document-creation-information.md#6.3), [7.2](package-information.md#7.2), [8.2](file-information.md#8.2).<br>where `<single line of text>` is the intended usage of the package assumed by the package supplier.<br>where `NONE` can be used to explicitly indicate there are NO other "usage scope".<br>where `NOASSERTION` can be used to explicitly indicate it is not clear if there are "usage scope" that may apply or not. |
```

Comment:
Perhaps this is different from what I was thinking in my comment above, about these being fields on packages. This appears to be set up where the "Usage Scope" field is a reference to an SPDX element that is defined somewhere else -- maybe even in a different SPDX document. Is that the intention?

If so, would this whole "usage section" work better instead as another subcategory of [Annotations](https://spdx.github.io/spdx-spec/annotations/), since that already exists as a way to add additional information on a separately-defined SPDX element? This appears to be partially copying some of the concepts from Annotations, but in a fully separate section.